### DISCUSSION TITLE
Payout

### DISCUSSION THREAD

**Discussion by n4ze3m** [community/community#76727] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I have waited 60 days for my first payout. On the GitHub payout page, it shows Nov 22 (today is Nov 23), and on Stripe, it shows Dec 22. Is there any issue, or do I need to wait another 30 days apart from the initial 60 days ? Thanks

**Comment by mjimenez98** [comment, marked as answer]
Hello @n4ze3m 👋 I am part of the Sponsors team.

All members of GitHub Sponsors go through a 60 day probation period before the first payout is made. This period begins the day you receive your first sponsorship, which in your case was Sep 20, 2023. Meanwhile, payout runs happen on a monthly basis, with the payment initiated in order for funds to land in your account around the 22nd of each month.

Your probation period officially ended on Nov 19, 2023 but this was after the payout run for November had already started, unfortunately. This means that your first payout will come in December.

On the plus side, payouts will happen monthly from then onwards. You can read further detail on this here: [4.3. Payment Timing.](https://docs.github.com/site-policy/github-terms/github-sponsors-additional-terms#43-payment-timing)

I’m sorry I don’t have better news to share today, but please let me know if I can help with anything else.

**Reply by n4ze3m** [reply]
Thank you for the clarification
